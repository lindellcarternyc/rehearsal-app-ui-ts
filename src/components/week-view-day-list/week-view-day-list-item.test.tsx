import * as React from 'react'
import * as renderer from 'react-test-renderer'

import WEEKS from '../../mocks/data/weeks'

const didClick = jest.fn()

import WeekViewDayListItem, { WeekViewDayListItemProps } from './week-view-day-list-item'
describe('WeekViewDayListItem', () => {
  it('renders without rehearsals', () => {
    const day = WEEKS[0].days[0]
    const props: WeekViewDayListItemProps = {
      day,
      didClick
    }
    const item = renderer.create(<WeekViewDayListItem {...props} />)
    expect(item).toMatchSnapshot()
  })

  it('renders with rehearsals', () => {
    const day = WEEKS[1].days[2]
    const props: WeekViewDayListItemProps = {
      day, didClick
    }
    const item = renderer.create(<WeekViewDayListItem {...props} />)
    expect(item).toMatchSnapshot()
  })
})