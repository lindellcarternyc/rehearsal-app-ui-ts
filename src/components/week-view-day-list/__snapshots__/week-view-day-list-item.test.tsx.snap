// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekViewDayListItem renders with rehearsals 1`] = `
<div
  className="item"
  data-value={undefined}
  onClick={[Function]}
  role="listitem"
>
  <div
    className="header"
  >
    Tue, January 16
  </div>
  <div
    className="ui bulleted list"
    role="list"
  >
    <div
      className="item"
      data-value={undefined}
      onClick={[Function]}
      role="listitem"
    >
      19:00 - 22:00
    </div>
  </div>
</div>
`;

exports[`WeekViewDayListItem renders without rehearsals 1`] = `
<div
  className="item"
  data-value={undefined}
  onClick={[Function]}
  role="listitem"
>
  <div
    className="header"
  >
    Sun, January 7
  </div>
  <p
    style={
      Object {
        "paddingLeft": "1rem",
      }
    }
  >
    No Rehearsals Today
  </p>
</div>
`;
